packages/sx05re/emuelec/bin/emuelecRunEmu.sh

```diff
@@ -498,6 +503,9 @@ fi
 
 emuelec-utils end_app_video
 
+# Kill MIDI Processes
+emuelec-utils killall_midi
```

Comment:
it is needed to set back midi_output = "OFF" in retroarch.cfg . I see that current behaviout is keeping the lastedt midi_output used and that's not ok. The midi_ouput must come back to midi_output = "OFF" @Langerz82